Usman.sol

```diff
@@ -0,0 +1,173 @@
+pragma solidity ^0.4.16;
+
+library SafeMath{
+    function add (uint256 a,uint256 b) internal returns (uint256){
+       uint c = a+b;
+        assert(c >= a);
+        return c;
+    }
+    
+    function sub(uint256 a,uint256 b) internal returns (uint c){
+         assert(b <= a);
+            c = a-b;
+    }
+    function mul(uint a,uint b) internal returns (uint){
+       uint c = a*b;
+        assert(a == 0 || c / a == b);
+        return c;
+    }
+    function div(uint a,uint b) internal returns (uint c){
+        c = a/b;
+    }
+}
+
+
+contract Ownable {
+    address public owner;
+
+
+    function Ownable() {
+        owner = msg.sender;
+    }
+
+    modifier onlyOwner() {
+        require(msg.sender == owner);
+        _;
+    }
+
+}
+
+
+contract ERC20Basic {
+    uint256 public totalSupply;
+    function balanceOf(address who) constant returns (uint256);
+    function transfer(address to, uint256 value);
+    event Transfer(address indexed from, address indexed to, uint256 value);
+}
+
+
+contract ERC20 is ERC20Basic {
+    function allowance(address owner, address spender) constant returns (uint256);
+    function transferFrom(address from, address to, uint256 value);
+    function approve(address spender, uint256 value);
+    event Approval(address indexed owner, address indexed spender, uint256 value);
+}
+
+
+contract BasicToken is ERC20Basic, Ownable {
+    using SafeMath for uint256;
+
+    mapping(address => uint256) balances;
+
+   
+    function transfer(address _to, uint256 _value) {
+        balances[msg.sender] = balances[msg.sender].sub(_value);
+        balances[_to] = balances[_to].add(_value);
+        Transfer(msg.sender, _to, _value);
+    }
+
+    
+    function balanceOf(address _owner) constant returns (uint256 balance) {
+        return balances[_owner];
+    }
+}
+
+
+contract StandardToken is ERC20, BasicToken {
```

Comment:
Please merge ERC20Basic,ERC20, BasicToken, StandardToken into one and name it ERC20